cypress/integration/buy-shirt.spec.ts

```diff
@@ -1,21 +1,32 @@
-describe("Buy a t-shirt", () => {
-  it("then the t-shirt should be bought", () => {
-    cy.visit("http://automationpractice.com/");
-    cy.get("#block_top_menu > ul > li:nth-child(3) > a").click();
-    cy.get("#center_column a.button.ajax_add_to_cart_button.btn.btn-default").click();
-    cy.get("[style*=\"display: block;\"] .button-container > a").click();
-    cy.get(".cart_navigation span").click();
+import {
+  // eslint-disable-next-line max-len
+  MenuContentPage, ProductListPage, ShopingCartPage, LoginPage, AddressPage, ShippingPage, PaymentPage, VerifyPage,
+} from "../page/index";
 
-    cy.get("#email").type("aperdomobo@gmail.com");
-    cy.get("#passwd").type("WorkshopProtractor");
-    cy.get("[id=SubmitLogin]").click();
-    cy.get(".cart_navigation button").click();
-    cy.get("[id=cgv]").click();
-    cy.get("[name=processCarrier]").click();
-    cy.get("p.payment_module > a.bankwire").click();
-    cy.get("p.cart_navigation > button.button.btn.btn-default.button-medium").click();
+const menuContentPage = new MenuContentPage();
+const productListPage = new ProductListPage();
+const shopingCartPage = new ShopingCartPage();
+const loginPage = new LoginPage();
+const addresPage = new AddressPage();
+const shippingPage = new ShippingPage();
+const paymentPage = new PaymentPage();
+const verifyPage = new VerifyPage();
 
-    cy.get("#center_column > div > p > strong")
-      .should("have.text", "Your order on My Store is complete.");
+describe("Buy a t-shirt", () => {
+  it("then should be bought a t-shirt", () => {
+    menuContentPage.visitMenuContentPage();
+    menuContentPage.goToTShirtMenu();
+    productListPage.addTShirtToCart();
+    shopingCartPage.checkoutPopUpEvent();
+    shopingCartPage.checkoutSummaryEvent();
+    loginPage.fillEmailField();
+    loginPage.fillPasswordField();
+    loginPage.submitBtnEvent();
```

Comment:
These 3 methods could be just one called login or singIn, at that method should receive user and pass as parameters, in that way, the loginPage object will be reusable for any user and pass